dev-util/sourcetrail/sourcetrail-2019.4.102.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A free and open-source cross-platform source explorer"
+DESCRIPTION="Sourcetrail helps you get productive on unfamiliar source code"
+HOMEPAGE="https://www.sourcetrail.com/"
+COPYRIGHT="2019 Andreas Stallinger and others"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/CoatiSoftware/Sourcetrail/archive/2019.4.102.tar.gz"
+CHECKSUM_SHA256="74b0f11904e5faaead94f92319001d77e23d102c7370f63f0f83a624e46312fe"
+SOURCE_DIR=""
```

Comment:
If SOURCE_DIR is the same as the content of the archive it's not needed here, also an empty line before ARCHITECTURES ...